grassroots-backend/src/phone-canvass/PhoneCanvass.service.ts

```diff
@@ -1,18 +1,24 @@
-import { Injectable } from "@nestjs/common";
+import { Injectable, UnauthorizedException } from "@nestjs/common";
 import { PhoneCanvassEntity } from "./entities/PhoneCanvass.entity.js";
 import { EntityManager, EntityRepository } from "@mikro-orm/core";
 import {
   CreatePhoneCanvassRequestDTO,
   CreatePhoneCanvassResponseDTO,
+  PhoneCanvassAuthTokenResponseDTO,
   PhoneCanvassProgressInfoResponseDTO,
 } from "grassroots-shared/dtos/PhoneCanvass/PhoneCanvass.dto";
 import { ContactEntity } from "../contacts/entities/Contact.entity.js";
 import { PhoneCanvassToContactEntity } from "./entities/PhoneCanvassToContact.entity.js";
+import { TwilioService } from "./Twilio.service.js";
+import { VoidDTO } from "grassroots-shared/dtos/Void.dto";
 
 @Injectable()
 export class PhoneCanvassService {
   repo: EntityRepository<PhoneCanvassEntity>;
-  constructor(private readonly entityManager: EntityManager) {
+  constructor(
+    private readonly entityManager: EntityManager,
+    private twilioService: TwilioService,
```

Comment:
General model I'm trying to follow here is that the PhoneCanvasService relies on the TwilioService for anything that directly hits Twilio, so at some point we can mock out the TwilioService completely.